x/evidence/module.go

```diff
@@ -158,7 +158,9 @@ func (am AppModule) ExportGenesis(ctx sdk.Context) json.RawMessage {
 }
 
 // BeginBlock executes all ABCI BeginBlock logic respective to the evidence module.
-func (AppModule) BeginBlock(_ sdk.Context, _ abci.RequestBeginBlock) {}
+func (am AppModule) BeginBlock(ctx sdk.Context, req abci.RequestBeginBlock) {
+	BeginBlocker(ctx, req, am.keeper)
```

Comment:
what's the naming convention for `BeginBlock` vs `BeginBlocker`?

Reply:
I see the two used interchangeably. `BeginBlock` is the defacto nomenclature, but we use `BeginBlocker` through the SDK in certain places. We should clean this up -- in a separate PR of course.